pkg/provisioners/bicep_test.go

```diff
@@ -154,3 +154,40 @@ func TestBicepReadProvisionerInputs(t *testing.T) {
 		})
 	}
 }
+
+func TestBicepInitializeStep(t *testing.T) {
+	type test struct {
+		name         string
+		templatePath string
+		want         string
+	}
+	tests := []test{
+		{
+			name:         "same",
+			templatePath: "testdata/bicep/inittest.bicep",
+			want: `foobar
+`,
```

Comment:
Is this backtick supposed to be on a new line?

Reply:
Ha! Yeah, one of the pre-commit hooks is a new line fixer, and it wants a blank line at the end of each file. We are comparing against the copied file, so the "want" includes the newline.